tests/ui/test.py

```diff
@@ -0,0 +1,102 @@
+import logging
+import os
+
+from playwright.sync_api import Page
+from pytest_bdd import given, parsers, scenarios, then, when
+from ui.llm_studio_page import LLMStudioPage
```

Comment:
Could you change this line (and also other imports) a relative import, i.e. `from .llm_studio_page import`?